### PR TITLE
Fix `TypeError: Cannot read property 'className' of undefined`

### PR DESCRIPTION
**Pull request by yamafaktory**
We are using jest-glamor-react in production in our company and so far, this is a super piece of code. Thanks for your awesome work 👍!

**What**:

I've recently (first time ever) faced a weird issue in some of our (non open-sourced) components:

```shell
TypeError: Cannot read property 'className' of undefined

      221 |     inputAA.simulate('change')
      222 |     wrapper.update()
    > 223 |     expect(wrapper).toMatchSnapshot()
          |                     ^
      224 |     expect(wrapper.find('input#a-property-a').prop('checked')).toBeTruthy()
      225 |     // Check the mock function calls after saving.
      226 |     wrapper

      at getSelectorsFromProps (node_modules/jest-glamor-react/dist/utils.js:44:25)
      at node_modules/jest-glamor-react/dist/utils.js:35:14
          at Array.reduce (<anonymous>)
      at getSelectors (node_modules/jest-glamor-react/dist/utils.js:31:16)
      at Function.print (node_modules/jest-glamor-react/dist/serializer.js:38:23)
      at printPlugin (node_modules/pretty-format/build/index.js:287:16)
      at printer (node_modules/pretty-format/build/index.js:332:12)
      at plugin.serialize.plugin.print.valChild (node_modules/pretty-format/build/index.js:289:23)
      at Object.print (node_modules/enzyme-to-json/createSerializer.js:22:14)
      at printPlugin (node_modules/pretty-format/build/index.js:287:16)
      at prettyFormat (node_modules/pretty-format/build/index.js:485:16)
      at Object.throwingMatcher (node_modules/expect/build/index.js:316:33)
      at Object.<anonymous> (src/components/Channels/ChannelSelector.test.jsx:223:21)
```

The error is thrown only because of the snapshot creation. Removing the `expect(wrapper).toMatchSnapshot()` line fixes it (but this obviously not what we want).

We are using Enzyme and all the other tests are passing (i.e. nothing is broken). The issue was triggered after splitting one component in smaller chunks.

**Why**:

For some reason, the `props` are undefined in the `getSelectorsFromProps` method (utils). Consequently, an error is thrown in https://github.com/kentcdodds/jest-glamor-react/blob/master/src/utils.js#L37-L38 when trying to access the `className` property of the `props` object.

**How**:

The changes involved are quite simple, returning an empty array early actually solves it. The unit tests of the repository are passing and our tests are not altered at all (a good sign!).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@ dist
 .opt-in
 .opt-out
 .DS_Store
+.eslintcache
```
Comment (yamafaktory) [inline .gitignore]: Sorry for the noise, but it was missing in the .gitignore file 😄.

**Review by kentcdodds — DISMISSED** [review]
Sounds good 👍

**Comment by kentcdodds** [comment]
Looks like we need a test for this. Could you add one?

**Comment by yamafaktory** [comment]
@kentcdodds Thanks a lot for the quick feedback 🚀🎉. I've added a small test case for that weird corner case.

**Review by kentcdodds — APPROVED** [review]
Super-duper. Thanks!

**Comment by yamafaktory** [comment]
@kentcdodds Thanks a ton ❤️!
